azure-functions/acbs-function/src/functions/activity-update-facility-fixed-fee.js

```diff
@@ -0,0 +1,88 @@
+/**
+ *
+ * Facility fixed fee `amount` amendment DAF
+ * *****************************************
+ * This DAF (Durable Activity Function) is never invoked directly.
+ * It is invoked via DOF (Durable Orchestrator Function).
+ *
+ * Pre-requisites
+ * --------------
+ * 0. 'npm install durable-functions'
+ * 1. Durable  HTTP trigger function (acbs-http)
+ * 2. Durable Orchestrator function (DOF) (acbs-amend-facility)
+ *
+ * ------------------
+ * HTTP -> DOF -> DAF
+ * ------------------
+ */
+
+const df = require('durable-functions');
+const api = require('../../api');
+const { findMissingMandatory } = require('../../helpers/mandatoryFields');
+const { getNowAsIsoString } = require('../../helpers/date');
+const { isHttpErrorStatus } = require('../../helpers/http');
+
+/**
+ * Handles the amendment of a facility fixed fee record in the ACBS system.
+ *
+ * This function performs the following operations:
+ * 1. Validates the input payload.
+ * 2. Checks for missing mandatory fields in the ACBS facility fixed fee input.
+ * 3. Submits the amendment to the ACBS system.
+ * 4. Handles the response from the ACBS system and returns the result.
+ *
```

Comment:
We've removed the other typings/docs here and kept this, is there a reason for this?

Reply:
Good point, have refactored and made docstring consistent across all DAF.